app/views/taxons/edit.html.erb

```diff
@@ -1,12 +1,13 @@
 <%= display_header title: taxon.title, breadcrumbs: [:taxons, taxon] %>
 
-<%= form_for taxon, url: taxon_path(taxon.content_id), method: "patch" do |f| %>
-  <%= f.text_field :title %>
+<%= simple_form_for taxon, url: taxon_path(taxon.content_id), method: :patch do |f| %>
   <%= f.hidden_field :content_id, value: taxon.content_id %>
   <%= f.hidden_field :base_path, value: taxon.base_path %>
-  <%= f.select :parent_taxons,
-    options_for_select(taxons_for_select, taxon.parent_taxons),
-    { include_blank: true },
-    { class: "select2", multiple: true } %>
+
+  <%= f.input :title %>
+
+  <%= f.input :parent_taxons, collection: taxons_for_select,
+      input_html: { class: :select2, multiple: true, include_blank: true } %>
+
   <%= f.submit "Save", class: "btn btn-lg btn-primary" %>
```

Comment:
We do, otherwise the button becomes a small gray rectangle instead of the large blue button.

<img width="193" alt="screen shot 2016-08-10 at 14 20 33" src="https://cloud.githubusercontent.com/assets/136777/17554986/9eedbd46-5f05-11e6-8262-bcdd7d50249c.png">